articles/java/sdk/java-sdk-azure-get-started.md

```diff
@@ -45,40 +45,14 @@ Which gives you a reply in the following format:
 }
 ```
 
-Next, copy the following into a text file on your system:
-
-```text
-# sample management library properties file
-subscription=ssssssss-ssss-ssss-ssss-ssssssssssss
-client=cccccccc-cccc-cccc-cccc-cccccccccccc
-key=aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa
-tenant=tttttttt-tttt-tttt-tttt-tttttttttttt
-managementURI=https\://management.core.windows.net/
-baseURL=https\://management.azure.com/
-authURL=https\://login.windows.net/
-graphURL=https\://graph.windows.net/
-```
-
-Replace the top four values with the following:
-
-- subscription: use the *id* value from `az account show` in the Azure CLI 2.0.
-- client: use the *appId* value from the output taken from a service principal output.
-- key: use the *password* value from the service principal output.
-- tenant: use the *tenant* value from the service principal output.
-
-Save this file in a secure location on your system where your code can read it. You may use this file for future code so it's recommended to store it somewhere external to the application in this article.
+Next, configure the environment variables with the following:
 
-Set an environment variable `AZURE_AUTH_LOCATION` with the full path to the authentication file in your shell.
+- <code>AZURE_SUBSCRIPTION_ID</code>: use the *id* value from `az account show` in the Azure CLI 2.0.
+- <code>AZURE_CLIENT_ID</code>: use the *appId* value from the output taken from a service principal output.
+- <code>AZURE_CLIENT_SECRET</code>: use the *password* value from the service principal output.
+- <code>AZURE_TENANT_ID</code>: use the *tenant* value from the service principal output.
 
-```bash
-export AZURE_AUTH_LOCATION=/Users/raisa/azureauth.properties
-```
-
-If you're working in a windows environment, add the variable to your system properties. Open a PowerShell window with administrator privleges and, after replacing the second variable with the path to your file, enter the following command:
-
-```powershell
-setx AZURE_AUTH_LOCATION "C:\<fullpath>\azureauth.properties" /m
-```
+For more approaches of authentication support, please refer to [Azure Identity](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/identity/azure-identity#azure-identity-client-library-for-java).
```

Comment:
"For more approaches of authentication support" seems not fit here. suggest "For more options of authentication" or "For support on authentication".

Reply:
Updated by `For more options of authentication`.